### PR TITLE
edit profile api

### PR DESCRIPTION
**Pull request by Ramkrishna-egov**
Edit profile api integrated in model,repo,provider format

### CODE REVIEW

**Comment by Ramkrishna-egov** [comment]
Edit profile api done, Search profile api call added in side bar, two Edit profile files were there in the project, deleted one

File: frontend/mgramseva/lib/screeens/Profile/EditProfile.dart [hunk 1]
```diff
@@ -27,15 +31,53 @@ class _EditProfileState extends State<EditProfile> {
...
-    userProvider.getUserProfileDetails({
-      "tenantId": "pb",
-      "id": [92],
-      "mobileNumber": "8004375123"
-    });
+    var editProvider = Provider.of<UserEditProfileProvider>(context, listen: false);
+    editProvider.getEditUser();
+  }
+
+  saveInputandedit(context, editUserChanges, EditUser editUser) async {
```
Comment (naveen-egov) [inline frontend/mgramseva/lib/screeens/Profile/EditProfile.dart:38]: Why is the User details being Hard coded

File: frontend/mgramseva/lib/screeens/Profile/EditProfile.dart [hunk 2]
```diff
@@ -18,6 +19,9 @@ class EditProfile extends StatefulWidget {
 }
 
 class _EditProfileState extends State<EditProfile> {
+  var name = new TextEditingController();
```
Comment (naveen-egov) [inline frontend/mgramseva/lib/screeens/Profile/EditProfile.dart:22]: What is the purpose of this input controllers here

File: frontend/mgramseva/lib/utils/Constants/I18KeyConstants.dart [hunk 3]
```diff
@@ -19,14 +22,53 @@ class Common {
...
+  String get VENDOR_NAME => 'CORE_EXPENSE_VENDOR_NAME';
+  String get EXPENSE_TYPE => 'CORE_EXPENSE_TYPE_OF_EXPENSES';
+  String get AMOUNT => 'CORE_EXPENSE_AMOUNT';
+  String get BILL_ISSUED_DATE => 'CORE_EXPENSE_BILL_ISSUED_DATE';
+  String get BILL_PAID => 'CORE_EXPENSE_BILL_PAID';
+  String get AMOUNT_PAID => 'CORE_EXPENSE_AMOUNT_PAID';
+  String get ATTACH_BILL => 'CORE_EXPENSE_ATTACH_BILL';
+}
+
+class CreateConsumer{
```
Comment (naveen-egov) [inline frontend/mgramseva/lib/utils/Constants/I18KeyConstants.dart:54]: Consumer should not have a prefix of Core

**Review by naveen-egov — COMMENTED** [review]
Kindly check the review and update accordingly

**Pull request merged**